sklearn/ensemble/_forest.py

```diff
@@ -1330,6 +1330,7 @@ class RandomForestRegressor(ForestRegressor):
         are "squared_error" for the mean squared error, which is equal to
         variance reduction as feature selection criterion, and "absolute_error"
         for the mean absolute error.
+        Training using the "mae" criterion is significantly slower than "mse".
```

Comment:
Hi @UnixJunkie 

It is indeed the case generally.
```python
In [1]: from sklearn.ensemble import RandomForestRegressor

In [2]: from sklearn.datasets import make_regression

In [3]: X, y = make_regression(n_samples=1000)

In [4]: %timeit RandomForestRegressor(criterion="mse").fit(X, y)
3.55 s ± 1.3 s per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [5]: %timeit RandomForestRegressor(criterion="mae").fit(X, y)
34.8 s ± 1.04 s per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

Here is a small suggestion taking the renaming into account:
```suggestion
        Training using "absolute_error" (i.e "mae") is significantly slower
        than when using "squared_error" (i.e "mse").
```

If you are personally interested in speeding up the training, you can have a look at [this section of the documentation regarding parallelization](https://scikit-learn.org/stable/modules/ensemble.html#parallelization).